src/compiler/Restler.Compiler/Dictionary.fs

```diff
@@ -58,8 +58,18 @@ type MutationsDictionary =
                 m |> Map.tryFind entryName
             | None -> None
 
+        member x.getRequestTypePayloadPrefix endpoint (method:string) = 
+            sprintf "%s/%s/" endpoint (method.ToLower())
+
+        member x.getRequestTypePayloadName endpoint (method:string) propertyNameOrPath = 
+            sprintf "%s/%s/%s" endpoint (method.ToLower()) propertyNameOrPath
+
+        member x.isRequestTypePayloadName endpoint (method:string) (propertyValue:string) = 
+             let prefix = sprintf "%s/%s" endpoint (method.ToLower())
```

Comment:
Should `prefix` include a trailing `/` like it does here in `getRequestTypePayloadPrefix`?

```
        member x.getRequestTypePayloadPrefix endpoint (method:string) = 
            sprintf "%s/%s/" endpoint (method.ToLower())
```

Could there be an issue here with `propertyValue.StartsWith(prefix)` incorrectly matching with a `prefix` that's actually a substring of `propertyValue`?

prefix = "foo/bar"
propertyValue = "foo/bar2"